apps/explorer/tests/utils/localnet.ts

```diff
@@ -25,16 +25,14 @@ export async function mint(address: string) {
     });
     const signer = new RawSigner(keypair, provider);
 
+    const coins = await provider.getCoins({ owner: address });
+    const coin_id = coins.data[0].coinObjectId;
+
     const tx = new Transaction();
     tx.moveCall({
-        target: '0x2::devnet_nft::mint',
-        arguments: [
-            tx.pure('Example NFT'),
-            tx.pure('An example NFT.'),
-            tx.pure(
-                'ipfs://bafkreibngqhl3gaa7daob4i2vccziay2jjlp435cf66vhono7nrvww53ty'
-            ),
-        ],
+        target: '0x2::pay::split_coin',
```

Comment:
Curious why we're doing this as a move call vs split + transfer commands.

Reply:
Good point, I'll get back to this when I am done with the rust side